### DISCUSSION TITLE
Test 6-all-crazy-wikipedia.

### DISCUSSION THREAD

**Discussion by Cristian1754** [IIC2133-PUC/2024-1#93] [Tarea 1] [answered]
Para el test 6-all-crazy-wikipedia tambien se va a contar la regla de los 10 segundos?. El resto de mis test se hacen todos en menos de 1 segundo pero ese test en particula dura 1 minuto 30. Queria saber si se va a tener otra clase de consideración con ese test.

Saludos!

**Comment by benjavicente** [comment, marked as answer]
Hola! Sí.

Eso sí, subimos test cases nuevos que probablemente te bajen el tiempo. Descárgalos, revísalos y coméntanos si efectivamente te baja el tiempo. Mañana mandaremos más información al respecto.
